WISE-Community/WISE-Client
# chore(Grading Tool): Convert StudentGradingTools

## Changes

- Converted StudentGradingTools from AngularJS to Angular.
- Removed some style classes from elements in the StudentGradingTools template that didn't seem to be needed anymore. You may want to double check to make sure I didn't remove something that we still need in terms of styling.

## Test

1. Go to the Grading Tool for a run
2. Go to the Grade By Team view
3. Select a workgroup
4. Make sure the workgroup select drop down works like before
- Next workgroup
- Previous workgroup
- Jump to workgroup
- Search for workgroup
- Make sure switching periods properly filters workgroups

Closes #745
